lib/modules/manager/mint/readme.md

```diff
@@ -0,0 +1,16 @@
+### Renovate supports Mintfile updates
+
+- [yonaskolb/Mint](https://github.com/yonaskolb/Mint)
+
+**note**
+
```

Comment:
```suggestion
```

Drop the `**note**` part, because the readme is short enough for users to read it in one go fully.